src/main-fuse.cpp

```diff
@@ -294,6 +294,10 @@ int hfs_listxattr(const char* path, char* buffer, size_t size)
 		for (const std::string& str : attrs)
 			output.insert(output.end(), str.c_str(), str.c_str() + str.length() + 1);
 
+        if ( buffer == NULL ) {
```

Comment:
This file is indented with tabs, not 8 spaces, and conditions are formatted `(like this)`, not `( like this )`.

Reply:
Sorry, my editor convert tabs, I guess. "( like this )" this is habit. Makes it easier to read.